CryptoPrices/Sources/MyCoin/Sources/MyCoin/MyCoinView/CoinDetailItem.swift

```diff
@@ -0,0 +1,50 @@
+//
+//  CoinDetailItem.swift
+//  MyCoin
+//
+//  Created by Khanh on 02/01/2023.
+//
+
+import DomainTestHelpers
+import Entities
+import Foundation
+
+public struct CoinDetailItem: Identifiable, Equatable {
```

Comment:
Just curious, what is the main purpose for making this struct conform to the `Identifiable` protocol? Do we really need to make the struct instances to be unique as there is just a single data object at a time and it simply comes from the server? 🤔

Reply:
Good question 🙏 . We don't need to make this struct conform to the `Identifiable`, I think it's more fit when using with List or stuff like that. Thank you for the raise :D. I removed it at https://github.com/nimblehq/nimble-crypto-ios/pull/81/commits/9d5cdb559bffa05ee3ae8db469423daf0ae46f90.